packages/slate-editor/src/modules/rich-formatting-menu/RichFormattingMenu.tsx

```diff
@@ -236,14 +229,25 @@ export function RichFormattingMenu({
                     onSubSuperScript={handleSubSupClick}
                     onFormatting={handleFormattingChange}
                     onLink={handleLinkButtonClick}
-                    // features
-                    withBoldFormat={!isInsideTableHeader}
+                    // text style
+                    withBold={!isInsideTableHeader}
+                    withItalic
+                    withUnderline
+                    // formatting
+                    withFormatting={
+                        formatting.active.includes(HeadingRole.TITLE) ||
+                        formatting.active.includes(HeadingRole.SUBTITLE)
+                            ? 'readonly'
+                            : true
+                    }
                     withAlignment={withAlignment}
-                    withBlockquotes={withBlockquotes && !isInsideTable && !isTitleOrSubtitleHeading}
-                    withHeadings={withHeadings && !isInsideTable && !isTitleOrSubtitleHeading}
-                    withInlineLinks={withInlineLinks && !isTitleOrSubtitleHeading}
-                    withLists={withLists && !isTitleOrSubtitleHeading}
-                    withParagraphs={withParagraphs && !isTitleOrSubtitleHeading}
+                    withBlockquotes={withBlockquotes && !isInsideTable}
+                    withHeadings={withHeadings && !isInsideTable}
+                    withInlineLinks={withInlineLinks}
```

Comment:
This should not be enabled for Title and Subtitle.

Reply:
:+1:  

#435